build/task-package.js

```diff
@@ -16,19 +16,15 @@ const PLATFORM = os.platform();
 
 const IGNORE = [
   // Ignore build stuff
-  '/appveyor.yml',
-  '/branding($|/)',
-  '/build($|/)',
-  '/dist($|/)',
+  '/\/appveyor.yml',
+  '/\/branding($|/)',
+  '/\/build($|/)',
+  '/\/dist($|/)',
   '/electron($|/)',
-  '/gulpfile.js',
   '/README.md',
   '/scripts($|/)',
   '/test($|/)',
 
-  // Ignore source code
-  '/ui($|/)',
-
   // Ignore `/app` and `/shared` once we compile main process code
   // ahead of time, rather than during runtime
   // '/app($|/)',
```

Comment:
We are now compiling main process code ahead of time. Shouldn't this comment be updated?